content/trusted-content/official-images/using.md

```diff
@@ -0,0 +1,103 @@
+tags. The purpose of this section is to show what image variants are available.
+
+![Example: supported tags for Ubuntu](../images/supported_tags.webp)
+
+Tags listed on the same line all refer to the same underlying image. Multiple
+tags can point to the same image. For example, in the previous screenshot taken
+from the `ubuntu` Docker Official Images repository, the tags `24.04`,
+`noble-20240225`, `noble`, and `devel` all refer to the same image.
+
+The `latest` tag for a Docker Official Image is often optimized for ease of use
+and includes a wide variety of useful software, such as developer and build tools.
+Because of their ease of use and wide applicability, `latest` images are often
+used in getting-started guides, but they're typically not optimized for production use.
+
+## Slim images
+
+A number of operating system and language stacks such as
+[Debian](https://hub.docker.com/_/debian/),
+[Node.js](https://hub.docker.com/_/node/),
+[Python](https://hub.docker.com/_/python/), and
+[Ruby](https://hub.docker.com/_/ruby/) have `slim` tag variants
+designed to provide a lightweight, production-ready base image
+with fewer packages.
+
+A typical consumption pattern for `slim`
+images is as the base image for the final stage of a
+[multi-staged build](https://docs.docker.com/build/building/multi-stage/).
+For example, you build your application in the first stage of the build
+using the `latest` variant and then copy your application into the final
+stage based upon the `slim` variant. Here is an example `Dockerfile`.
+
+```dockerfile
+FROM node:latest AS build
+WORKDIR /app
+COPY package.json package-lock.json ./
+RUN npm ci
+COPY . ./
+FROM node:slim
+WORKDIR /app
+COPY --from=build /app /app
+CMD ["node", "app.js"]
+```
+
+## Alpine
+
+Many Docker Official Images repositories also offer `alpine` variants. These
+images are built on top of the [Alpine Linux](https://www.alpinelinux.org/)
+distribution rather than Debian or Ubuntu. Alpine Linux is focused on providing
+a small, simple, and secure base for container images, and Docker Official
+Images `alpine` variants typically aim to install only necessary packages. As a
+result, Docker Official Images `alpine` variants are typically even smaller
+than `slim` variants.
+
+It's worth keeping in mind that Alpine Linux is based on musl libc, as opposed
+to glibc, which is used by most other Linux distributions. This difference in C
+implementations may influence how your program runs, depending on the type of
+application you're building. With Alpine Linux, your program can't use
+dynamically linked glibc objects, as those aren't available on musl. To run
+your programs on Alpine Linux, you can:
+
+- Compile your program against musl libc
+- Statically link glibc libraries into your program
+- Avoid C dependencies altogether (for example, build Go programs without CGO)
+- Install a glibc compatibility layer manually, alongside musl
```

Comment:
We should definitely without a doubt drop this last bullet point (see references linked from https://github.com/docker-library/official-images/issues/10794).

I also wonder if we should work in more of the content from our standard "Alpine blurb" here? https://github.com/docker-library/docs/blob/c5a57f3ec8904a2232986c1bd18f45558482eee1/.template-helpers/variant-alpine.md (maybe one of the links?  maybe a reference to where to find this information on Hub, because it sometimes changes per-image, with `golang:alpine` being a notable example: https://github.com/docker-library/docs/blob/c5a57f3ec8904a2232986c1bd18f45558482eee1/golang/variant-alpine.md)

Reply:
Included some of the wordings from there. I prefer not to link out to the orange website or issues on other projects if I can help it.